openeo-geotrellis/src/main/scala/org/openeo/geotrellis/package.scala

```diff
@@ -219,6 +222,55 @@ package object geotrellis {
       })
   }
 
+  /**
+   * Python equivalent: health_check_extent
+   */
+  def healthCheckExtent(projectedExtent: ProjectedExtent)(implicit logger: Logger): Boolean = {
+    // positive width and height is already enforced in Extent.
+    val horizontal_tolerance = 1.1
+    val polygonIsUTM = projectedExtent.crs.proj4jCrs.getProjection.getName == "utm"
+    if (polygonIsUTM) {
+      // This is an extent that has the highest sensible values for northern and/or southern hemisphere UTM zones
+      val utmProjectedBoundsOriginal = Extent(166021.44, 0000000.00, 833978.56, 10000000)
+      val utmProjectedBounds = utmProjectedBoundsOriginal.buffer(
+        utmProjectedBoundsOriginal.width * horizontal_tolerance, 0)
+      if (!projectedExtent.extent.intersects(utmProjectedBounds)) {
+        logger.warn("healthCheckExtent dangerous extent: " + projectedExtent)
```

Comment:
Make this logging more informative towards openEO user. Same as above.
If the purpose of the health check is to avoid using a 'dangerous' extent, than we may also want to avoid alarming the user.